tardis/model/base.py

```diff
@@ -171,23 +185,30 @@ def __init__(
         self._abundance = abundance
         self.time_explosion = time_explosion
         self._electron_densities = electron_densities
+        self.raw_abundance = self._abundance
+        self.raw_isotope_abundance = isotope_abundance
         v_outer = self.velocity[1:]
         v_inner = self.velocity[:-1]
         density = (
             self.homologous_density.calculate_density_at_time_of_simulation(
                 self.time_explosion
             )[self.v_boundary_inner_index + 1 : self.v_boundary_outer_index + 1]
         )
+
+        raw_abundance = self.raw_abundance.copy()
+        raw_abundance["mass_number"] = -1
+        raw_abundance = raw_abundance.set_index([raw_abundance.index, "mass_number"])
+        isotope_abundance = raw_abundance.append(self.raw_isotope_abundance)
```

Comment:
This should not be decayed, should it?

```suggestion
        isotope_abundance = raw_abundance.append(self.raw_isotope_abundance.decay(500))
```